requirements/app/base.txt

```diff
@@ -13,3 +13,4 @@ inquirer>=2.10.0
 psutil<5.9.4
 click<=8.1.3
 lightning_api_access>=0.0.3
+aiohttp>=3.8.0, <=3.8.3
```

Comment:
This shouldn't be a base requirement. Usage of this component is optional.

Reply:
I asked the same question and @lantiga suggests adding it to the base. @nohalon Could you confirm here?

Reply:
yes @lantiga suggested we can add this req to this file

Reply:
I would still suggest a solution like the one done by @hhsecond in #15844

Reply:
I fully agree! @lantiga @nohalon Can you confirm?

Reply:
yes let's do that! @akihironitta

Reply:
the case was that in past lightning did not allow install any extra which was fixed in #15793

Reply:
@Borda I wasn't aware that there was no extra options before. Thank you for fixing the extra distribution 💜 I'll start adding `aiohttp` to `requirements/app/components.txt` to enable users to install it with `pip install lightning[components]`, but let me know if you (or someone else) have other suggestions :)